cmd/ovirt-populator/ovirt-populator.go

```diff
@@ -168,10 +187,16 @@ func loadEngineConfig(secretName, engineURL, namespace string) engineConfig {
 		klog.Fatal(err.Error())
 	}
 
+	insecure, err := strconv.ParseBool(string(secret.Data["insecureSkipVerify"]))
```

Comment:
that's going to be change after the populator PR, it changed to use a mounted volume with the secret. it will be required to read the relevant file within the pod.

Reply:
waiting for the new changes, I"ll adjust it accordingly